Plots/Scatter/NCL_scatter_3.py

```diff
@@ -0,0 +1,90 @@
+"""
+NCL_scatter_3.py
+================
+This script illustrates the following concepts:
+    - Drawing a scatter plot over a map
+    - Drawing markers on a map
+    - Attaching markers to a map
+    - Changing the marker color and size in a map plot
+    - Drawing markers on a map indicating the locations of station data
+   
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/scatter_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/scatter_3_1_lg.png and
+                        https://www.ncl.ucar.edu/Applications/Images/scatter_3_2_lg.png
```

Comment:
If you write the plot URLs like this, the Sphinx formatting is a bit off. I have found that is is better just to leave it as one long line. It will show up okay on the website